File: uri/1 - Basics/1011/Main.java
import java.util.*;

public class Main{

	public static void main(String args[]){
		Scanner scanner = new Scanner(System.in);
	    int a = scanner.nextInt();
	    System.out.format("VOLUME = %.3f\n", (4.0/3.0)*3.14159* Math.pow(a,3));
	}

}

File: uri/1 - Basics/1011/Main.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var a int
	fmt.Scanf("%d", &a)
	fmt.Printf("VOLUME = %.3f\n", (4.0/3.0)*3.14159*math.Pow(float64(a), 3))
}
